client/src/templates/Introduction/components/super-block-intro.tsx

```diff
@@ -1,21 +1,48 @@
 import React from 'react';
+import { graphql, useStaticQuery } from 'gatsby';
 import { useTranslation, Trans } from 'react-i18next';
-import { Alert, Spacer } from '@freecodecamp/ui';
+import { Alert, Spacer, Container, Row, Col } from '@freecodecamp/ui';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import { useFeatureIsOn } from '@growthbook/growthbook-react';
 import { SuperBlocks } from '../../../../../shared/config/curriculum';
 import { SuperBlockIcon } from '../../../assets/superblock-icon';
 import { Link } from '../../../components/helpers';
+import CapIcon from '../../../assets/icons/cap';
+import DumbbellIcon from '../../../assets/icons/dumbbell';
+import CommunityIcon from '../../../assets/icons/community';
+import { CompletedChallenge } from '../../../redux/prop-types';
+import { completedChallengesSelector } from '../../../redux/selectors';
+
+interface SuperBlockIntroQueryData {
+  challengeNode: {
+    challenge: {
+      fields: {
+        slug: string;
+      };
+    };
+  };
+}
 
 interface SuperBlockIntroProps {
   superBlock: SuperBlocks;
   onCertificationDonationAlertClick: () => void;
   isDonating: boolean;
+  completedChallenges: CompletedChallenge[];
 }
 
+const mapStateToProps = createSelector(
+  completedChallengesSelector,
+  (completedChallenges: CompletedChallenge[]) => ({
+    completedChallenges
+  })
+);
```

Comment:
```suggestion
const mapStateToProps = (state: unknown) => ({
  completedChallenges: completedChallengesSelector(
    state
  ) as CompletedChallenge[]
});
```
There's nothing to memoize since nothing is computed, so we can just use a normal selector.